packages/gafl-webapp-service/src/processors/__tests__/date-and-time-display.spec.js

```diff
@@ -34,6 +43,39 @@ describe('displayStartTime', () => {
     expect(moment.utc).toHaveBeenCalledWith(startDate, null, expect.any(String))
   })
 
+  it.each([[50], [10000], [0]])(
+    'expected text replaces the date string if start after payment is selected using correct constant',
+    async startAfterMinutes => {
+      constant.START_AFTER_PAYMENT_MINUTES = startAfterMinutes
+      const startDate = '2021-01-01T00:00:00.000Z'
+      const mssgs = getSampleRequest().i18n.getCatalog()
+      const expectedReturnText = constant.START_AFTER_PAYMENT_MINUTES + mssgs.licence_summary_minutes_after_payment
+      moment.utc.mockImplementation(() => ({
+        tz: () => ({
+          locale: jest.fn(() => ({ format: () => '' })),
+          format: () => {}
+        })
+      }))
+      const startAfterPaymentMinutes = displayStartTime(getSampleRequest(), { startDate, licenceToStart: 'after-payment' })
+      expect(startAfterPaymentMinutes).toEqual(expectedReturnText)
+    }
+  )
+
+  it('date in plain english is returned if start after payment is not selected', () => {
+    const startDate = '2021-01-01T00:00:00.000Z'
+    const expectedReturnDateValue = '0.00am (first minute of the day) on 1 January 2021'
+    const realMoment = jest.requireActual('moment-timezone')
+    moment.utc.mockReturnValueOnce(realMoment(startDate))
+    moment.utc.mockImplementation(() => ({
+      tz: () => ({
+        locale: jest.fn(() => ({ format: () => '' })),
+        format: () => {}
+      })
+    }))
+    const returnDateValue = displayStartTime(getSampleRequest(), { startDate, licenceToStart: 'not-after-payment' })
+    expect(returnDateValue).toEqual(expectedReturnDateValue)
+  })
```

Comment:
I don't think this is necessary - what does it test that the following test doesn't?